docs/get-started/cli.md

```diff
@@ -4,4 +4,219 @@ title: On your local machine
 
 # Get started on your local machine
 
-TODO
+Although it takes a little longer to get started, building Packs on your local machine has some advantages over the the web-based editor:
+
+- You can use your own code editing tools, such as [Visual Studio Code](https://code.visualstudio.com/).
+- You can use your own version control system, such as [GitHub](https://github.com).
+- You can use popular JavaScript libraries[^1], such as those in [NPM](https://www.npmjs.com/).
+
+[^1]: Not all libraries available on NPM are compatible with the Packs SDK. See the [Using libraries](../guides/advanced/libraries.md) guide for more information.
+
+Local development is enabled through the `coda` command line tool (CLI). Keep reading to learn how to install the CLI and use it to build a Pack.
+
+## Before you get started
+
+To create a Pack you will need a Coda account, with [Doc Maker access](https://help.coda.io/en/articles/3388781-members-and-roles) in your workspace. If you're new to Coda, [sign up](https://{{coda.domain}}/signup) for a free account and you'll automatically be made a Doc Maker in your personal workspace.
+
+Make sure you have `node` and `npm` available on your machine. These two tools often come bundled together, and more information on how to install them is available in the [npm docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+The instructions below assume some familiarity with the terminal / command prompt. If you aren't used to using this interface consult the help material for your operating system.
+
+## Set up the Pack structure
+
+1. Create a directory for your new Pack.
+
+    ```sh
+    mkdir my-pack
+    cd my-pack
+    ```
+
+1. Install the Packs SDK.
+
+    ```sh
+    npm install @codahq/packs-sdk
+    ```
+
+    The Packs SDK includes both the `coda` CLI as well as the libraries and type definitions needed to build Packs.
+
+1. Create the file structure for your Pack.
+
+    ```sh
+    npx coda init
+    ```
+
+
+The `coda init` command creates the basic skeleton of a Pack based off of our [template Pack](https://github.com/coda/packs-examples/tree/main/examples/template). It's not required, but it's a fast way to get started.
+
+Your directory should now contain the following files:
+
+* `formulas.ts` - A place to define the Coda formulas, including those that power sync tables and column formats.
+* `manifest.ts` - Defines the metadata and contents of your Pack.
+* `node_modules` - The dependencies downloaded from NPM (standard for Node.js projects).
+* `package-lock.json` - The versions of the dependencies downloaded from NPM (standard for Node.js projects).
+* `package.json` - The project's dependencies from NPM (standard for Node.js projects).
+* `schemas.ts` - A place to define the schemas (structured data types) used by your Pack.
+* `types.ts` - A place to define TypeScript types for the data used by your Pack.
```

Comment:
As we go, feel free to suggest a better file layout if you like. This is what has worked best in the `packs` repo but that doesn't mean we have to recommend it externally.